modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/toast/commands/OpenToast.es.js

```diff
@@ -123,11 +115,13 @@ function openToast({
 			onClose({event});
 		}
 
-		if (!container || !containerId) {
-			rootElement.parentNode.removeChild(rootElement);
-		}
+		if (!event.defaultPrevented) {
```

Comment:
Hey @jbalsas, when opening the session toast and triggering another toast by typing on console `Liferay.Util.openToast({message: 'test'})`, an error will be thrown: 

![Screen Shot 2020-08-06 at 14 07 47](https://user-images.githubusercontent.com/7663513/89561088-61219300-d7ee-11ea-8bf8-2cbda3a58ff8.png)

I think we must just make sure We have `event` defined here

Reply:
Hey @diegonvs, that's a nice catch! I'm curious... did you actually see an error occur?

The only way I see this can happen is if you actually created an `onClick` handler that called the `onClose` method without passing an `event` (which is not the snippet you pasted, thus the curiosity)

```javascript
Liferay.Util.openToast({
    message: ' message hey you', 
    onClick({event, onClose}) {onClose();} 
});
```

I'll if the thing off and manually push to Brian, see if we make it to the release window.